terraform/terraform.g4

```diff
@@ -81,9 +81,23 @@ val
    | IDENTIFIER index?
    | DESCRIPTION
    | filedecl
+   | functioncall
    | EOF_
    ;
 
+functioncall
+   : functionname '(' functionarguments ')'
```

Comment:
Hi, maybe it's better to use `?`, instead of empty line in `functionarguments`

Like this one
```
functioncall
   : functionname '(' functionarguments? ')'
   ;

functionarguments
   : expression (',' expression)*
   ;
 
```

Reply:
Sure.